### DISCUSSION TITLE
sluggish behaviour in Flatlist

### DISCUSSION THREAD

**Discussion by coucoseth** [react-native-tvos/react-native-tvos#373] [Q&A] [answered]
in android Tv, focus from one element to another inside a flatlist is really slower than users' action from the remote. How can this be solved

**Comment by douglowder** [comment, marked as answer]
It's difficult to answer this question without seeing your specific application. Certainly, there are apps built with this repo on Android TV that do not have this issue. If you can provide a sample application with source code that shows the problem, I will investigate.

**Reply by coucoseth** [reply]
Turns out i had alot of code in my app.js, i restructured my code into multiple components and its working just fine. thanks
